cegs_portal/search/views/v1/non_targeting_reos.py

```diff
@@ -15,8 +16,23 @@
 from cegs_portal.utils.pagination_types import Pageable
 
 
+def non_targeting_regulatory_effects_tsv(data):
```

Comment:
In the same way HTML templates are in the `templates` directory and JSON templates are in the `json_templates` directory, this should be in a `tsv_templates` directory

Reply:
Makes sense. Got it.